### PR TITLE
docs: operator based installation for the VPP dataplane

### PR DESCRIPTION
**Pull request by AloysAugustin**
## Description

This PR updates the Installation and APIServer CRD definitions from the current operator master.

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
VPP dataplane instructions now use operator-based install
```

### CODE REVIEW

File: calico/_includes/charts/tigera-operator/crds/operator.tigera.io_apiservers_crd.yaml [hunk 1]
```diff
@@ -46,3 +46,9 @@ spec:
     storage: true
     subresources:
       status: {}
+status:
```
Comment (caseydavenport) [inline calico/_includes/charts/tigera-operator/crds/operator.tigera.io_apiservers_crd.yaml:49]: I think this section should be removed, not sure why it gets added...

File: calico/getting-started/kubernetes/vpp/getting-started.md [hunk 2]
```diff
@@ -22,9 +22,9 @@ The VPP dataplane mode has several advantages over standard Linux networking pip
   * Reduces first-packet latency for packets to services
   * Preserves external client source IP addresses all the way to the pod
 
-The VPP dataplane is entirely compatible with the other {{site.prodname}} dataplanes, meaning you can have a cluster with VPP-enabled nodes along with regular nodes. This makes it easy to migrate a cluster from Linux or eBPF networking to VPP networking.
+The VPP dataplane is entirely compatible with the other {{site.prodname}} dataplanes, meaning you can have a cluster with VPP-enabled nodes along with regular nodes. This makes it possible to migrate a cluster from Linux or eBPF networking to VPP networking.
```
Comment (caseydavenport) [inline calico/getting-started/kubernetes/vpp/getting-started.md:25]: :rofl:

File: calico/getting-started/kubernetes/vpp/getting-started.md [hunk 3]
```diff
@@ -22,9 +22,9 @@ The VPP dataplane mode has several advantages over standard Linux networking pip
...
 
-In the future, the VPP dataplane will offer additional features for network-intensive applications, such as providing `memif` userspace packet interfaces to the pods (instead of regular Linux network devices).
+In the addition, the VPP dataplane offers some specific features for network-intensive applications, such as providing `memif` userspace packet interfaces to the pods (instead of regular Linux network devices), or exposing the VPP Host Stack to run optimized L4+ applications in the pods.
```
Comment (caseydavenport) [inline calico/getting-started/kubernetes/vpp/getting-started.md:27]: "In addition" rather than "In the addition"

**Review by caseydavenport — COMMENTED** [review]
Looking good - thanks @AloysAugustin! 

Just a couple comments. Would also be good to get a quick look from @lwr20

**Comment by AloysAugustin** [comment]
Thanks for the review! The latest commit should address your comments.

**Comment by lwr20** [comment]
LGTM.

**Review by lwr20 — APPROVED** [review]
LGTM.

**Pull request merged**